pages/builds/parallelizing_builds.md.erb

```diff
@@ -36,11 +36,11 @@ To run the same step over all 5 of the agents we can set the parallelism field f
 
```

Comment:
Totally! https://github.com/ArturT/knapsack#info-for-buildkitecom-users

That's how we parallelise our `rspec` tests.